lib/msf/ui/console/command_dispatcher/developer.rb

```diff
@@ -435,4 +433,16 @@ def cmd_time_help
     print @@time_opts.usage
     print_line
   end
+
+  private
+
+  def modified_files
+    # Using an array avoids shelling out, so we avoid escaping/quoting
+    changed_files = %w[git diff --name-only]
```

Comment:
Um isn't this meant to be like this? 
```suggestion
    changed_files = %w(git diff --name-only)
```

Though this avoided the array syntax.